BuildAndTest.proj

```diff
@@ -70,15 +64,15 @@
       <TestAssemblies Include="Binaries\Debug\Roslyn.Compilers.VisualBasic.CommandLine.UnitTests.dll" />
       <TestAssemblies Include="Binaries\Debug\Roslyn.Diagnostics.Analyzers.FxCop.UnitTests.dll" />
       <TestAssemblies Include="Binaries\Debug\Roslyn.Diagnostics.Analyzers.UnitTests.dll" />
+      <!--
       <TestAssemblies Include="Binaries\Debug\Roslyn.Services.CSharp.UnitTests.dll" />
       <TestAssemblies Include="Binaries\Debug\Roslyn.Services.UnitTests.dll" />
-      <TestAssemblies Include="Binaries\Debug\Roslyn.Services.VisualBasic.UnitTests.dll" />
+      <TestAssemblies Include="Binaries\Debug\Roslyn.Services.VisualBasic.UnitTests.dll" /> -->
     </ItemGroup>
 
-    <xunit
-      Assemblies="@(TestAssemblies)"
-      Html="UnitTestResults.html"
-      ShadowCopy="false" />
+    <exec
+      Command="packages\xunit.runners.2.0.0-rc1-build2826\tools\xunit.console.exe @(TestAssemblies, ' ') -noshadow -parallel all" />
```

Comment:
Perhaps this path should be made into a global property in one of the common targets files (the path to xUnit)...  Unrelated to this change, but I notice that even though we're using the 2.0 runner, we are still referencing xunit.1.9.2 from all of our test projects (perhaps this <Reference> should be in a common targets file as well).